tests/foreman/virtwho/ui/test_hyperv_sca.py

```diff
@@ -0,0 +1,120 @@
+"""Test class for Virtwho Configure UI
+
+:Requirement: Virt-whoConfigurePlugin
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: Virt-whoConfigurePlugin
+
+:Team: Phoenix
+
+:TestType: Functional
+
+:Upstream: No
+"""
+import pytest
+from fauxfactory import gen_string
+
+from robottelo.config import settings
+from robottelo.utils.virtwho import deploy_configure_by_command
+from robottelo.utils.virtwho import deploy_configure_by_script
+from robottelo.utils.virtwho import get_configure_command
+from robottelo.utils.virtwho import get_configure_file
+from robottelo.utils.virtwho import get_configure_id
+from robottelo.utils.virtwho import get_configure_option
+
+
+@pytest.fixture()
+def form_data():
```

Comment:
non-blocking:
```suggestion
def hyperv_form_data():
```

Maybe you can do that for esx,kubevirt,libvirt as well

Reply:
@adarshdubey-star Because we have five hypervisors, and every hypervisors have cli,api,ui form_data, we want to unify the format, if we change this , we will change the other files together, we will consider to update totally in the future